app/mailers/submission_mailer.rb

```diff
@@ -0,0 +1,9 @@
+class SubmissionMailer < ApplicationMailer
+
+  def contract(user)
+    @user = user
+    attachments["plik.pdf"] = File.read('plik.pdf')
```

Comment:
Tak to na pewno nie zadziała - bo niby gdzie jest ten plik? Na tę chwilę można zrobić tak jak w testach (proszę odnaleźć za pomocą polecenia `git grep plik.pdf` )